releasenotes/notes/runtime-config-change-06132b1d9b1a2bd9.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    Add the ability to change log_level at runtime.
```

Comment:
We should explain a bit more how or point to the official doc we if we have one. So users reading the CHANGELOG knows how to use it.

Reply:
Indeed the release note would benefit from being a little bit more verbose.